### ISSUE TITLE
Checklist header area does not have sufficient contrast

### ISSUE THREAD

**Issue by ericwbailey** [a11yproject/a11yproject.com#1100]
Reported via a private Slack.

Currently, the blue background image that reads "Checklist" placed over the green background color has a contrast ratio of `2.3:1`. We'll need to update it to be WCAG compliant.

**Comment by ericwbailey** [comment]
Closed via https://github.com/a11yproject/a11yproject.com/pull/1102.
